### PR TITLE
Docs: Make it clear how to integrate rule configs

### LINKED ISSUE #904: Improve rule documentation and make it more obvious how rule configurations can be integrated into the `.sonarwhalrc` file
It seems [it's not clear enough](https://twitter.com/jgarber/status/976505040537116674)

### PR DESCRIPTION
**Pull request by alrra**
## Pull request checklist

Make sure you:

- [x] Signed the [Contributor License Agreement](https://cla.js.foundation/sonarwhal/sonarwhal)
- [x] Followed the [commit message guidelines](https://sonarwhal.com/docs/contributor-guide/getting-started/pull-requests/#commit-messagess)

For non-trivial changes, please make sure you also:

- [x] Added/Updated related documentation.
- [ ] Added/Updated related tests.

## Short description of the change(s)

Improves the rule documentation and makes it more obvious how rule configurations can be integrated into the `.sonarwhalrc` file.

- - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -

Fix #904

### CODE REVIEW

**Comment by alrra** [comment]
FYI @jgarber623.

File: packages/rule-http-cache/README.md [hunk 1]
```diff
@@ -378,9 +389,10 @@ https://example.com/assets/12345/styles.css
 [caching best practices]: https://jakearchibald.com/2016/caching-best-practices/
 [google http caching]: https://developers.google.com/web/fundamentals/performance/optimizing-content-efficiency/http-caching
 [how well you know the web]: https://youtu.be/vAgKZoGIvqs?t=12m20s
-[maxage0]: http://httparchive.org/about.php#maxage0
 [maxage0 image]: https://chart.googleapis.com/chart?chd=t:-1%7C49,51,100,100,51,51,51,51,51,51,100,50,50,50,50,50,50,50,51,51,51,50,50,50&chxl=0:%7C+%7C12%2F16%7C+%7C1%2F17%7C+%7C2%2F17%7C+%7C3%2F17%7C+%7C4%2F17%7C+%7C5%2F17%7C+%7C6%2F17%7C+%7C7%2F17%7C+%7C8%2F17%7C+%7C9%2F17%7C+%7C10%2F17%7C+%7C11%2F17&chxt=x&chs=600x300&cht=lxy&chco=184852&chxs=0,676767,11.5,0,lt,676767&chxtc=0,8&chm=N**+%,184852,0,1::2,12,,h::8&chds=0,100,0,100&chts=184852,24&chtt=Cacheable+Resources&chls=2&chma=5,5,5,25
+[maxage0]: http://httparchive.org/about.php#maxage0
```
Comment (molant) [inline packages/rule-http-cache/README.md]: Why did you change the order? If it's sorted alphabetically should `maxage0` be above `maxage0 image`?

Comment (alrra) [reply packages/rule-http-cache/README.md]: >  If it's sorted alphabetically should maxage0 be above maxage0 image?

At least for `vim`, `the space from (from `[maxage0 image]`) comes before before `]` (from `[maxage0]`), hence this change.

Comment (molant) [reply packages/rule-http-cache/README.md]: But we should be sorting by the content inside the brackets, without taking them into account.
`maxage0` goes before than `maxage0 image`.
